src/NHSD.BuyingCatalogue.Ordering.Domain/Common/ValueObject.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace NHSD.BuyingCatalogue.Ordering.Domain.Common
+{
+    public abstract class ValueObject

```

Comment:
a summary on these classes to explain what they're for would be nice

Reply:
👍